search-app/docs/api.md

```
@@ -59,6 +59,7 @@ See the [CMR Client Partner User Guide](https://wiki.earthdata.nasa.gov/display/
     * [Science keywords](#c-science-keywords)
     * [TwoD coordinate system](#c-twod-coordinate-system)
     * [Collection data type](#c-collection-data-type)
+    * [Collection progress](#c-collection-progress)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)